operations/utility_storage/Troubleshoot_Ceph_New_RGW_Deployment_Failing.md

```diff
@@ -0,0 +1,128 @@
+# Troubleshoot Ceph New RGW Deployment Failing
+
+Troubleshoot an issue where a new RGW deployment is failing because of an address already in use. This bug corrupts the health of the Ceph cluster.
+
+Return the Ceph cluster to a healthy state by resolving issues with failing deployment.
+
+## Prerequisites
+
+This procedure requires admin privileges.
+
```

Comment:
I would reorganize the document by adding a section to verify the error.

```
## Verify Ceph is deploying a second RGW deployment
<add your ceph -W cephadm output from below>

I would also suggest running the command `ceph orch ps --deamon_type rgw`. Check that the rgw deployment running has a different name than the one attempting to deploy as observed by `ceph -W cephadm`.

If this is the problem being observed, continue to the next section to resolve this error.

## Procedure to resolve two RGW deployments
```

Reply:
Just made these changes, let me know if it all looks good now :)